README.md

```diff
@@ -15,40 +15,79 @@ To run this project locally, follow these steps:
 
 Clone the repository to your local machine. Replace `YOUR_USERNAME` with your GitHub username:
 
-git clone https://github.com/YOUR_USERNAME/Nc-news-project.git
+```bash
+git clone https://github.com/YOUR_USERNAME/your-repository-name.git
+```
 
 ### 2. **Create new repository on Git and set a remote branch**
 
 Get a remote branch. Replace `YOUR_NEW_REPO_URL_HERE` with the URL of your new repo in Git.
 
+```bash
 cd path to the clone project
 git init
 git remote add origin YOUR_NEW_REPO_URL_HERE
+```
 
-### 2. **Create the env. Files**
+### 3. **Create the env. Files**
 
-Create `.env.development` and `.env.test` files with:
+Create `.env.development` and `.env.test` files in the root directory of your project with the following content:
 
 PGDATABASE=nc_news
 PGDATABASE=nc_news_test
```

Comment:
I would hide this from the readme and point them towards the SQL file to look for the database names